.github/workflows/multi-platform-docker-build.yml

```diff
@@ -0,0 +1,110 @@
+name: crtsh-exporter-multi-platform-docker-build
+
+on:
+  push:
+    # tags:
+    #   - "v*.*.*" 
+    branches:
+      - master
+
+  schedule:
+    # This will run at 11:00 PM UTC every Monday.
+    - cron: '0 23 * * 1'
+jobs:
+  crtsh-exporter-multi-platform-docker-build:
+    name: crtsh-exporter
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY: "ghcr.io" 
+      REPO: "${{ github.repository_owner }}/crtsh-exporter" 
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set SHA_SHORT env var and get Kernel version
+        id: vars
+        run: |
+          echo "SHA_SHORT=$(git rev-parse --short HEAD)" >> ${GITHUB_ENV}
+          echo "VERSION=$(uname --kernel-release)" >> ${GITHUB_ENV}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Login
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GHCR }}
+          
+      - name: Buildx linux/amd64 Docker Image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64
```

Comment:
Could `platforms` not be run as a matrix (loop)?

Reply:
I tested this on my own personal github account, and using the parameter `platforms: linux/arm64/v8,linux/amd64,linux/arm64` to have parallel runs would timeout on me. If you have GA Runners or Pro account, I could change them to run in parallel

Reply:
I don't. Let's then leave as-is. Thanks for the explanation.